tests_integration/test_data/projects/snowpark_glob_patterns/requirements.txt

```diff
@@ -0,0 +1 @@
+snowflake-snowpark-python
syrupy

```

Comment:
Is this intentional or something weird with `\n` in this file?

Reply:
Something weird